fs/fs_zenfs.cc

```diff
@@ -879,6 +963,11 @@ Status ZenFS::DecodeSnapshotFrom(Slice* input) {
     files_.insert(std::make_pair(zoneFile->GetFilename(), zoneFile));
     if (zoneFile->GetID() >= next_file_id_)
       next_file_id_ = zoneFile->GetID() + 1;
+
+    for (uint32_t i = 0; i < zoneFile->GetNrLinks(); i++) {
+      std::shared_ptr<ZoneFile> z = zoneFile;
+      files_.insert(std::make_pair(z->GetLinkFilename(i), z));
```

Comment:
```suggestion
      files_.insert(std::make_pair(z->GetLinkFilename(i), zoneFile));
```
I think there is no reason to make a temporary `shared_ptr` here.

Reply:
Yeah, removed the temp variable.